src/client/src/database.rs

```diff
@@ -139,20 +139,20 @@ impl Database {
 
     async fn handle(&self, request: Request) -> Result<u32> {
         let mut client = self.client.make_database_client()?.inner;
-        let request = self.to_rpc_request(request);
+        let request = self.to_rpc_request(request, None);
         let response = client.handle(request).await?.into_inner();
         from_grpc_response(response)
     }
 
     #[inline]
-    fn to_rpc_request(&self, request: Request) -> GreptimeRequest {
+    fn to_rpc_request(&self, request: Request, trace_id: Option<u64>) -> GreptimeRequest {
```

Comment:
How about using `QueryContext` here? We will soon be having span_id

Reply:
https://github.com/shuiyisong/greptimedb/blob/8a66f72c90a6bd4e4ff3cde5af4664d4ec27573b/src/query/src/dist_plan/merge_scan.rs#L140

Reply:
I see. Perhaps we will be relying #2016 for future